WcaOnRails/app/views/competitions_mailer/notify_users_of_results_presence.html.erb

```diff
@@ -0,0 +1,11 @@
+<p>
+  The results of <%= link_to @competition.name, competition_url(@competition) %> you've competed recently are now available.
```

Comment:
I think we could make the phrasing here a little less awkward. How about something like:

```
The results of <competition name> (which you competed at) have just been posted.
```

As you said, we don't need to agonize over the contents of this email *too* much, as we can easily change it later =)